### PR TITLE
Remove unused gems and plugin

### PR DESCRIPTION
**Pull request by maya**
We don't use redcarpet, rouge, or this redcarpet extender plugin, so this removes them.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,12 +2,10 @@ source 'https://rubygems.org'
...
 gem 'json'
-gem 'redcarpet'
 gem 'open-uri-cached'
 gem 'jekyll-redirect-from'
 gem 'octopress-autoprefixer'
 
-gem 'rouge', '1.9'
 gem 'scss_lint', require: false
 
 gem 'jekyll-compose', group: [:jekyll_plugins]
```
Comment (hursey013) [inline Gemfile]: @maya Do you use `jekyll-compose` for anything?

Comment (maya) [reply Gemfile]: not sure what it's for or if I'm using it.

Comment (hursey013) [reply Gemfile]: Looks like its a tool for writing new posts and associating different states with them (published, unpublished, draft, etc). Looks like we are just adding new posts with regular jekyll way, so it should be safe to remove if we are looking to remove anything that's unused.

Comment (maya) [reply Gemfile]: Do you know what the `, group: [:jekyll_plugins]` does on the end of it?

Comment (hursey013) [reply Gemfile]: That just automatically includes the plugin in jekyll, it saves you a step of having to also include the name of the plugin in the `_config.yml`.

**Pull request merged**